src/www/model/events.ts

```diff
@@ -14,18 +14,16 @@
 
 import {Server} from './server';
 
-export interface OutlineEvent {}
+export interface OutlineEvent {
```

Comment:
We shouldn't force Outline events to have a server field. We need an alternative solution.